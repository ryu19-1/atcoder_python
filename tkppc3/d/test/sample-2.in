5 5 5
1 6 15 15 7
14 11 11 5 14
3 5 5 5
2 3 3 4
1 5 3 5
4 3 4 4
2 2 4 3
