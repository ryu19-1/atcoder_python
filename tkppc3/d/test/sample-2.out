98
54
140
-90
0
